.github/actions/validate-repo-version/action.yml

```diff
@@ -0,0 +1,64 @@
+name: Validate Repo Versions
+description: Action that validates that the versions in `config/versions.json` match existing versions in the repository
+inputs:
+  repos-to-check:
+    type: string
+    required: true
+    description: Space-separated list of ACCESS-NRI repositories to check
```

Comment:
There's really two supported repos for these checks? If someone mis-spells something or adds a gibberish repo to check, I guess nothing wrong would happen (except for the outputs not being set?) Not sure whether this should be a note in the description or a step with a quick check

Reply:
This has been reworked to fail when the repo it is given is not in the `versions.json` :+1: